build.sbt

```diff
@@ -8,21 +8,22 @@ lazy val `play-mailer` = (project in file("."))
     scalaVersion := scala212,
     crossScalaVersions := Seq(scala211, scala212)
   )
-    
+
 val PlayVersion = playVersion("2.6.0-RC2")
 
 libraryDependencies ++= Seq(
-  "com.typesafe.play" %% "play" % PlayVersion % Provided,
+  "com.google.inject" % "guice" % "4.0", // 4.0 to maybe make it work with 2.5 and 2.6
```

Comment:
hm, yeah I think that might prolly be a good idea. I will definitly look into it.
I guess I can just do it like that:
```
sudo: false
jdk: oraclejdk8
language: scala
matrix:
  include:
  - env: PLAY_VERSION=2.5.15
  - env: PLAY_VERSION=2.6.0-RC2
script: sbt +test
```

and then read the PlayVersion from env? And maybe make a comment that the travis needs to be changed (for tests) if another play version comes in ?